cypress/support/commands.ts

```diff
@@ -23,3 +33,20 @@ Cypress.Commands.add('useSubscriptionStatus', (condition) => {
         cy.reload()
     }
 })
+
+Cypress.Commands.add('pollUntilPresent', (eventName: string, timeout = 5000): Cypress.Chainable<CaptureResult[]> => {
```

Comment:
i really don't get Cypress and it's so slow I didn't run this locally, but it'll be something like this ultimately

Reply:
but for sure we have to poll in posthog-js in both cypress tests we used to have, and playwright tests we have now